Mapping/ClassMetadataFactory.php

```diff
@@ -312,7 +312,7 @@ private function getOrCreateAttribute(ClassMetadata $classMetadata, $attributeNa
                     ) {
                         $link = true;
                         foreach ($relationSerializerMetadata->getAttributesMetadata() as $attributeMetadata) {
-                            if (1 >= count(array_intersect($normalizationGroups, $attributeMetadata->getGroups()))) {
+                            if (1 <= count(array_intersect($normalizationGroups, $attributeMetadata->getGroups()))) {
```

Comment:
Fixed yesterday in 92f5fab3d3e56ce360fbfc35eda0fd69a6ddbf17. Can you remove this change?